en/advanced_config/pid_tuning_guide_fixedwing.md

```diff
@@ -121,8 +121,10 @@ The overall softness / hardness of the control loop can be adjusted by
 the time constant. The default of 0.5 seconds should be fine for normal
 fixed wing setups and usually does not require adjustment.
 
-- [FW_ATT_TC](../advanced_config/parameter_reference.md#FW_ATT_TC) - set to a default of 0.5 seconds, increase to make the
-  response softer, decrease to make the response harder.
+- [FW_P_TC](../advanced_config/parameter_reference.md#FW_P_TC) - set to a default of 0.5 seconds, increase to make the

```

Comment:
Great to have these fixes, but would have been better in separate PR.

Reply:
:-) Reasonable assumption, but not the case. The errors are non-blocking. Never the less I will be working my way and clearing through all the link errors in the near future.